src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingDiagnosticAnalyzer.cs

```diff
@@ -19,7 +19,6 @@ internal sealed class RenameTrackingDiagnosticAnalyzer : DiagnosticAnalyzer, IBu
         internal const string RenameToPropertyKey = "RenameTo";
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(DiagnosticDescriptor);
-        public bool OpenFileOnly(Workspace workspace) => true;
```

Comment:
so at some point, we want to move open file only analyzers to OOP as well. but this one shouldn't move even at that point since it require rename service which will not exist in OOP side.

Reply:
How is that controlled?  Thanks!

Reply:
@CyrusNajmabadi It's described at the top of this post.